CredentialsManager/FilesExceptions/MyFileNotFoundException.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace CredentialsManager.FilesExceptions
+{
+  public class MyFileNotFoundException : Exception
+  {
+    public MyFileNotFoundException()
+    {
+    }
+
+    public MyFileNotFoundException(string details) : base(details)
```

Comment:
Prefer...
```
public MyFileNotFoundException(string file) : base($"{file} file not found ore not able to open!")
{

}
```

Reply:
Also, you can make a static method named `FilenameToExceptionMessage` if you don't like putting that long string in the base constructor call.

Reply:
I updated it in [493cd86](https://github.com/csinn/CSharp-From-Zero-To-Hero-v2/pull/43/commits/493cd86aa678a20b0c535f9fbea6a18198e61285)